src/ConnectedMachine/docs/Install-AzConnectedMachinePatch.md

```diff
@@ -0,0 +1,435 @@
+---
+external help file:
+Module Name: Az.ConnectedMachine
+online version: https://learn.microsoft.com/powershell/module/az.connectedmachine/install-azconnectedmachinepatch
+schema: 2.0.0
+---
+
+# Install-AzConnectedMachinePatch
+
+## SYNOPSIS
+The operation to install patches on a hybrid machine identity in Azure.
+
+## SYNTAX
+
+### InstallExpanded (Default)
+```
+Install-AzConnectedMachinePatch -Name <String> -ResourceGroupName <String> -MaximumDuration <String>
+ -RebootSetting <VMGuestPatchRebootSetting> [-SubscriptionId <String>]
+ [-LinuxParameterClassificationsToInclude <VMGuestPatchClassificationLinux[]>]
+ [-LinuxParameterPackageNameMasksToExclude <String[]>] [-LinuxParameterPackageNameMasksToInclude <String[]>]
+ [-WindowParameterClassificationsToInclude <VMGuestPatchClassificationWindows[]>]
+ [-WindowParameterExcludeKbsRequiringReboot] [-WindowParameterKbNumbersToExclude <String[]>]
+ [-WindowParameterKbNumbersToInclude <String[]>] [-WindowParameterMaxPatchPublishDate <DateTime>]
+ [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### Install
```

Comment:
"Not-Expanded" parameter sets can be removed.

Reply:
changed in the latest commit